driver-ui/src/main/java/eu/tsystems/mms/tic/testframework/pageobjects/internal/AbstractUiElementList.java

```diff
@@ -30,12 +30,13 @@
 public abstract class AbstractUiElementList<SELF extends UiElementBase> implements UiElementList<SELF> {
     private final SELF uiElement;
     private int iteratorIndex = 0;
```

Comment:
Well, if you remove the default initialization of the other `int` fields you can also do that for `iteratorIndex`